distributions/otelcol-contrib/manifest.yaml

```diff
@@ -208,6 +208,7 @@ connectors:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector v0.105.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/exceptionsconnector v0.105.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/grafanacloudconnector v0.105.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/otlpjsonconnector v0.105.0
```

Comment:
I've copied this to https://github.com/open-telemetry/opentelemetry-collector-releases/pull/615/files#diff-c830bcf08de8b5a2d3f89d4dea5d1ce025f069b272e17236edd25296198a3b34R211